File: unity/Assets/Scripts/Multiplayer/Matchmaking/MatchmakingRemoveResponse.cs
using Michitai.Multiplayer.Errors;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Michitai.Multiplayer.Matchmaking
{
    /// <summary>
    /// Response returned when a matchmaking lobby is successfully removed.
    /// Can only be called by the host before the game starts.
    /// </summary>
    [System.Serializable]
    public class MatchmakingRemoveResponse : ApiResponse<EMatchmakingRemoveError>
    {
        /// <summary>
        /// Confirmation message for the lobby removal.
        /// </summary>
        public string message;
    }
}

File: java/src/main/java/com/michitai/multiplayer/matchmaking/MatchmakingRemoveResponse.java
package com.michitai.multiplayer.matchmaking;

import com.fasterxml.jackson.annotation.JsonProperty;
import com.michitai.multiplayer.ApiResponse;

/**
 * Response returned when a matchmaking lobby is removed.
 */
public class MatchmakingRemoveResponse extends ApiResponse {
    @JsonProperty("message")
    private String message;

    public String getMessage() {
        return message;
    }

    public void setMessage(String message) {
        this.message = message;
    }
}
